src/lib/routing/file.ts

```diff
@@ -0,0 +1,9 @@
+import type { FileLinkFragment } from '@lib/types/datocms';
```

Comment:
I'm only using this in a single place right now, so could've just copied it there. In the No Dodos website I improved the routing and started a [`lib/routing/`](https://github.com/No-Dodos/nododos-website/tree/main/src/lib/routing) setup which I hope we can reuse here. So this is a really small start for that :)